ruby_review_week_four.rb

```diff
@@ -1,37 +1,129 @@
 # You may NOT use Google. These questions are just to test where you are at. Get through as many of these as you can. If you don’t get through all of them it’s ok - they are designed to take longer than the available time. If you can’t remember a bit of code, write the pseudo-code that represents what you would write your Ruby code.
 
 # 1. What is a variable?
+A variable is a value that can change depending on the conditions passed through it by the rest of the code
 
 # 2. What Ruby method can you use to make a loop from an array?
+for.each
 
 # 3. In Ruby, what are booleans?
+Boolean values are true or false statements
 
 # 4. How do classes relate to objects?
+Classes are used to create objects, classes are an efficent way to make an amount of objects easily thats only limited by the written code.
 
 # 5. What is an instance variable?
+An instance variable is a variable that is inside the scope of a class
 
 # 6. Choose an everyday item (cup, computer, person) and give it two instance variables (which will be attributes), and one method (to describe a function of the object). Code the initialize method in full (for these attributes only), and the definition line of the instance method (but do not code the entire method).
+class Computer
+  def initialize(screen_size, os)
+    @screen_size = screen_size
+    @os = os
+  end
+  def description
+    p "The screen size is #{@screen_size} and the operating system is #{@os}"
```

Comment:
this is great! careful with following instructions carefully, you didn't need to code the instance method in full